qa/integration/pom.xml

```diff
@@ -233,6 +233,15 @@
             <artifactId>kapua-translator-kura-jms</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.kapua</groupId>
+            <artifactId>kapua-sso-api</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
Please define the version of this artifact on the root `pom.xml` in the `dependencyManagement` section and use in here only `groupId` and `artifactId`

Reply:
So the line `<version>1.2.0-SNAPSHOT</version>` should then be in file kapua\qa\pom.xml ?

Reply:
On the root pom, like you did! 👍